Agenda for the technical planning meeting

florianziemen opened the pull request.

andrewgettelman commented:
1. I'm not sure we need a half hour for all the tools. Maybe we can just do 15 minutes (10 min presentation, 5 min questions) and have more time for discussion. Presentations and/or links could be provided in advance for us to poke around with them.  I'm not sure where we draw the line at the 'core infrastructure' (healpix, protocol, catalogs) and the other analysis tools (Uxarray, Pythia, easy.gems). We might need to engage with the 'tracker' crowd (TEMPESTextremes, PYFLXTRKR, etc). Maybe a summary of that. It's the specific tools maybe we can shorten, and maybe we need discussion after presenting the protocol, healpix, zarr and catalogs (maybe those are shorter too)?
2. I think starting with a goals discussion perhaps in the introduction would be good. This might go over the ideas about global datasets we are storing and then regional data.  It might also talk about science questions and how that leads us to particular data sets. We haven't had much of that yet: so maybe it starts here.
3. Looks like the 2nd day is more technical. We might want to start with storage needs, then processing and environments. That will include perhaps discussing tutorials for analysis packages and making sure they are installed in environments. 
4. Somewhere we might talk about the suggested agenda for the week (you have examples, but maybe a Q&A) and then also how we might hand off analysis codes between nodes and how that would work.

florianziemen commented:
Thanks @andrewgettelman for thinking through things and the comments following from this.

I've tried to accommodate them. We now have day 1 for setting and data access and day 2 for the analysis.
Drawing the line between catalog and uxarray seems a bit arbitrary, but day 2 is less busy than day 1 so far, and I think the catalog is something that everybody will use and the tools for actually reading the data then are up to the individual...

I have added a slot on the trackers, but I myself have no contacts in that direction. If you (or somebody else) has ideas for who could present on that, that would be appreciated. I know that in EERIE they used pyeddytracker, and that was quite a headache because it relies on some outdated matplotlib version for finding the contours of the eddies...

andrewgettelman commented:
Thanks @florianziemen , I can try to get someone to speak about trackers.

florianziemen commented:
So, I've updated the schedule and also added a link to the registration.
See https://github.com/digital-earths-global-hackathon/digital-earths-global-hackathon.github.io/blob/tech-meeting/hosting/technical/meeting-2024-10-21.md
for the preview.

florianziemen commented:
discussed during today's meeting.
